Jenkinsfile

```diff
@@ -65,8 +65,12 @@ pipeline {
       steps {
         xvnc(useXauthority: true) {
           sh """
-            ./full-build.sh --tp=${selectedTargetPlatform()} \
-              ${javaVersion() == 17 ? '' : '--toolchains releng/toolchains.xml -Pstrict-release-jdk'}
+            mvn -f org.eclipse.xtext.full.releng --batch-mode clean deploy \
+              ${javaVersion() == 17 ? '' : '--toolchains releng/toolchains.xml -Pstrict-release-jdk'} \
+              -PuseJenkinsSnapshots \
```

Comment:
The full-build script also defined `-Dit-archetype-tests-skip=true`, but that is actually not necessary because that variable is initialized with true.

The same applies for the full-deploy script inlined below.